src/orqa.cpp

```diff
@@ -0,0 +1,71 @@
+#include "orqa.h"
+
+GENERIC_CRC8 ghst_crc(GHST_CRC_POLY);
+
+Orqa::Orqa()
+{
+}
+
+void Orqa::Init()
+{
+    delay(VRX_BOOT_DELAY);
+}
+
+void Orqa::SendIndexCmd(uint8_t index)
+{
+    uint8_t band = GetBand(index);
+    uint8_t channel = GetChannel(index);
+    uint8_t currentGHSTChannel = GHSTChannel(band, channel);
+    uint16_t currentFrequency = GetFrequency(index);
+    SendGHSTUpdate(currentFrequency, currentGHSTChannel);
+}
+
+
+void Orqa::SendGHSTUpdate(uint16_t freq, uint8_t ghstChannel)
+{
+    uint8_t packet[] = {
+        0x82, 0x0C, 0x20,                                       // Header
```

Comment:
My only comment is to make some of these header bytes defines.  But Im not fussed until we start to have multiple packets e.g. DVR start/stop.